web/init/src/Ship.jsx

```diff
@@ -6,16 +6,30 @@ import { configureStore } from "./redux";
 import PropTypes from "prop-types";
 
 import "./scss/index.scss";
+const bodyClass = "ship-init";
 
-export const Ship = ({ apiEndpoint }) => (
-  <div id="ship-init-component">
-    <Provider store={configureStore(apiEndpoint)}>
-      <AppWrapper>
-        <RouteDecider />
-      </AppWrapper>
-    </Provider>
-  </div>
-);
+export class Ship extends React.Component {
+  componentDidMount() {
+    document.body.classList.add(bodyClass);
```

Comment:
Just leaving a note here that React components should avoid modifying the global document or using unexpected global styles that could conflict with other components.

For now this is fine but it will have to be a temporary fix.

Reply:
Issue: https://github.com/replicatedhq/ship/issues/572